libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -57,7 +55,8 @@ def __init__(self,
     super(GoogleCloudCompute, self).__init__(self.project_id)
 
   def Instances(self,
-                refresh: bool = True) -> Dict[str, 'GoogleComputeInstance']:
+                refresh: Optional[bool] = True
+               ) -> Dict[str, 'GoogleComputeInstance']:
```

Comment:
Probably some more yapf stuff

Reply:
so these ones, to avoid adding the long-line exception thing, I actually moved it to a new line otherwise it is line too long

Reply:
Ah, I see, nevermind then :)